src/main/java/com/esri/samples/portal/token_authentication/README.md

```diff
@@ -0,0 +1,40 @@
+<h1>Token authentication</h1>
+
+<p>access a map service that is secured with ArcGIS token-based authentication</p>
+
+<p><img src="TokenAuthentication.png"/></p>
+
+<h2>Use case</h2>
+
+<p>Applications often require accessing data from private map services on remote servers. A token authentication system can be used to allow app users who hold a valid username and password to access the remote service. </p>
+
+<h2>How to use the sample</h2>
+
+<p>When starting the sample, the user is presented with an authentication dialog for the ArcGIS Online map server. Upon successful authentication, access to the map server is granted and the data is displayed in the map view.</p>
```

Comment:
this section was recently refactored for the same sample on the Android Repo, I've made some edits to this section based on that.

"When starting the sample, the user is challenged for an ArcGIS Online login to view the protected map service. Enter a username and password for an ArcGIS Online named user account (such as your ArcGIS for Developers account). Upon successful authentication, the protected map service will display in the map."